### PR TITLE
docs: add "How to Build Shield Documentation"

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- add "How to Build Shield Documentation"

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [] User guide updated
- [] Conforms to style guide

### CODE REVIEW

File: admin/HowToBuildDocs.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# How to Build Shield Documentation
```
Comment (datamweb) [inline admin/HowToBuildDocs.md:1]: We use Material for MkDocs for implementation. It is better to add the description in this section that includes the line as follows.
https://squidfunk.github.io/mkdocs-material/getting-started/

File: admin/HowToBuildDocs.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+```consolse
+mkdocs build
+```
+
+## See the Docs
+
+```consolse
+mkdocs serve
+```
```
Comment (datamweb) [inline admin/HowToBuildDocs.md:26]: Command ` mkdocs gh-deploy` is also good for manual implementation if there is a problem. Of course, I think it needs administrator access.

File: admin/HowToBuildDocs.md
Comment (datamweb) [inline admin/HowToBuildDocs.md:1]: We have previously named the files as follows.
**how_to_build_docs.md**

File: admin/HowToBuildDocs.md (hunk 1 above)
Comment (kenjis) [reply admin/HowToBuildDocs.md]: Added.

File: admin/HowToBuildDocs.md (hunk 2 above)
Comment (kenjis) [reply admin/HowToBuildDocs.md]: Added.

File: admin/HowToBuildDocs.md
Comment (kenjis) [reply admin/HowToBuildDocs.md:1]: Done.

**Review by datamweb — APPROVED** [review]
Thank you! 🥀

**Pull request merged**